zproject/default_settings.py

```diff
@@ -708,3 +708,7 @@
 INPUT_COST_PER_GIGATOKEN: int = 0
 OUTPUT_COST_PER_GIGATOKEN: int = 0
 MAX_PER_USER_MONTHLY_AI_COST: float | None = 0.5
+
+# URL of the onboarding video displayed to new users.
+# Set it to None to disable it.
+ONBOARDING_VIDEO_URL: str | None = "/static/videos/onboarding-video.mp4"
```

Comment:
We'll update this URL as per our final decision in [#design > onboarding video: distribution mechanism](https://chat.zulip.org/#narrow/channel/101-design/topic/onboarding.20video.3A.20distribution.20mechanism/with/2105059)

Reply:
Done.